scenes/quests/story_quests/stella/1_stella_stealth/stella_stealth.tscn

```diff
@@ -203,26 +203,26 @@ y_sort_enabled = true
 tile_set = ExtResource("2_gylsq")
 metadata/_edit_lock_ = true
 
 [node name="TopGrass" type="TileMapLayer" parent="TileMapLayers"]
-tile_map_data = PackedByteArray("AAAmABUAAQADAAMAAAAmABYAAQAMAAAAAAAmABgAAQADAAMAAAAiABsAAQADAAMAAAAiABwAAQADAAMAAAAiAB8AAQADAAMAAAAkAB8AAQALAAAAAAAjAB8AAQADAAMAAAApAB8AAQADAAMAAAAqACAAAQADAAMAAAAnAB8AAQADAAMAAAAhACQAAQADAAMAAAAdACQAAQADAAMAAAAcACQAAQAKAAAAAAAbACQAAQADAAMAAAAgACcAAQADAAMAAAAfACcAAQADAAMAAAAeACcAAQALAAAAAAAdACcAAQADAAMAAAAYACUAAQADAAMAAAAYACQAAQADAAMAAAAYACMAAQADAAMAAAAYACIAAQADAAMAAAAXACUAAQADAAMAAAAXACQAAQADAAMAAAAHABsAAQADAAMAAAAGABsAAQAKAAAAAAAFABsAAQADAAMAAAALABkAAQADAAMAAAALABgAAQADAAMAAAANABgAAQADAAMAAAAMABgAAQAKAAAAAAARABgAAQADAAMAAAAQABgAAQADAAMAAAATABgAAQADAAMAAAARACYAAQAKAAAAAAAPACcAAQAKAAAAAAANACYAAQADAAMAAAAKACYAAQADAAMAAAAHACMAAQADAAMAAAAHACIAAQADAAMAAAAKAA4AAQAKAAAAAAD//w4AAQAKAAAAAAACABQAAQADAAMAAAABABQAAQADAAMAAAD//xEAAQADAAMAAAD//xAAAQADAAMAAAD//w8AAQADAAMAAAD//wsAAQADAAMAAAADAAoAAQADAAMAAAAEAAoAAQADAAMAAAAKABIAAQADAAMAAAAKABMAAQADAAMAAAAKABQAAQADAAMAAAAJABQAAQADAAMAAAA=")
+tile_map_data = PackedByteArray("AAAmABUAAQADAAMAAAAmABYAAQAMAAAAAAAmABgAAQADAAMAAAAiABsAAQADAAMAAAAiABwAAQADAAMAAAAiAB8AAQADAAMAAAAkAB8AAQALAAAAAAAjAB8AAQADAAMAAAApAB8AAQADAAMAAAAqACAAAQADAAMAAAAnAB8AAQADAAMAAAAhACQAAQADAAMAAAAdACQAAQADAAMAAAAcACQAAQAKAAAAAAAbACQAAQADAAMAAAAgACcAAQADAAMAAAAfACcAAQADAAMAAAAeACcAAQALAAAAAAAdACcAAQADAAMAAAAYACUAAQADAAMAAAAYACQAAQADAAMAAAAYACMAAQADAAMAAAAYACIAAQADAAMAAAAXACUAAQADAAMAAAAXACQAAQADAAMAAAAHABsAAQADAAMAAAAGABsAAQAKAAAAAAAFABsAAQADAAMAAAALABkAAQADAAMAAAALABgAAQADAAMAAAANABgAAQADAAMAAAAMABgAAQAKAAAAAAARABgAAQADAAMAAAAQABgAAQADAAMAAAATABgAAQADAAMAAAARACYAAQAKAAAAAAAPACcAAQAKAAAAAAANACYAAQADAAMAAAAKACYAAQADAAMAAAAHACMAAQADAAMAAAAHACIAAQADAAMAAAD//xEAAQADAAMAAAD//xAAAQADAAMAAAD//w8AAQADAAMAAAADAAoAAQADAAMAAAAEAAoAAQADAAMAAAAKABIAAQADAAMAAAAKABMAAQADAAMAAAD//w0AAQAKAAAAAAAKAA0AAQAKAAAAAAA=")
 tile_set = ExtResource("2_gylsq")
 
 [node name="Player" parent="." instance=ExtResource("3_qvtgt")]
-position = Vector2(108, 369)
+position = Vector2(480, 703)
```

Comment:
I don't think you meant to commit this:

![image](https://github.com/user-attachments/assets/d2bd26fa-f7aa-4b81-9b02-64c237597f1c)